HermaFx.DataAnnotations.Tests/NotDefaultTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+using NUnit.Framework;
+using System;
+
+namespace HermaFx.DataAnnotations
+{
+	[TestFixture]
+	public class NotDefaultTest
+	{
+		private class Model
+		{
+			[NotDefault]
+			public Guid Value { get; set; }
+		}
```

Comment:
Add tests for "Guid?", along with tests for other value types (int, short, an struct, etc.)
Also add tests for 'string' or an object not of value type.